actors.md

```diff
@@ -265,14 +270,31 @@ func SlashConsensusFault(block1, block2 BlockHeader) {
 	self.Miners.Remove(miner)
 	self.UpdateStorage(-1 * miner.Power)
 
-	// Now delete the miner (maybe this is a bit harsh, but i'm okay with it for now)
+	// Remove the actor's state from the network entirely.
 	miner.SelfDestruct()
 }
 ```
 
+### SlashStorageFault
+
+> SlashStorageFault penalizes a storage miner for not submitting a PoSt within the required [time window](#TODO-link-to-faulty-submission). This may be called by anyone who detects the faulty behavior. The slashed miner loses all of their staked collateral and all of their power, and as a result, is no longer a candidate leader for extending the chain. See [faults](faults.md).
+
+Parameters: 
+
+- miner Address
+
+Return: None
+
+```go
+func SlashStorageFault(miner Address) {
+	// TODO
```

Comment:
+1, will file an issue, thanks

Reply:
ah, right. I *thought* I had written this already, its a method on the miner itself.

Reply:
It was initially on the storage market, but all it did was call through to a method on the miner, and didnt actually do anything on the storage market, so it was moved to the miner.

It definitely feels weird to have the consensus slashing here, and the storage slashing there though, so I could be convinced that it should be moved back. But we would then be adding network overhead for a small UX improvement.

Reply:
Ah, of course, thanks for spotting. I'll remove this.